src/psyclone/domain/gocean/transformations/raise_psyir_2_gocean_kern_trans.py

```diff
@@ -82,17 +107,18 @@ class KernTrans(Transformation):
         invalid.
 
     '''
+    VALID_NAME = re.compile(r'[a-zA-Z_][\w]*')
+
     def __init__(self, metadata_name):
         super().__init__()
 
-        pattern = re.compile(r'[a-zA-Z_][\w]*')
-        if not metadata_name or not pattern.match(metadata_name):
+        if not metadata_name or not \
+           RaisePSyIR2GOceanKernTrans.VALID_NAME.match(metadata_name):
             raise TransformationError(
-                f"Error in {self.name} transformation. The kern_trans "
-                f"transformation requires the name of the variable "
-                f"containing the metadata to be set to a valid value "
-                f"before applying the transformation, but found "
-                f"'{metadata_name}'.")
+                f"Error in {self.name} transformation. The "
+                f"raise_psyir_2_gocean_kern_trans transformation requires the "
```

Comment:
Please use the class name here, i.e. RaisePSyIR2GOceanKernTrans.

Reply:
Oh yes.